checkers/whyNoLint_checker.go

```diff
@@ -0,0 +1,52 @@
+package checkers
+
+import (
+	"go/ast"
+	"regexp"
+	"strings"
+
+	"github.com/go-lintpack/lintpack"
+	"github.com/go-lintpack/lintpack/astwalk"
+)
+
+func init() {
+	info := lintpack.CheckerInfo{
+		Name:    "whyNoLint",
+		Tags:    []string{"style", "experimental"},
+		Summary: "Ensures that //nolint comments include an explanation",
```

Comment:
```suggestion
		Summary: "Ensures that `//nolint` comments include an explanation",
```
wdyt?

Reply:
Sure. I hadn't realized this was meant to be markdown syntax, but it looks like there's plenty of precedent for this in other checkers. Should we add something to the GoDoc for `lintpack.CheckerInfo.Summary` to make that more clear for checker implementers? Would the expectation be for general-case markdown syntax, or just the backticks for fixed-width (all I've found examples of)?

Anyway, will add for this.

Reply:
It's not markdown-only, as `gocritic doc <checker>` gives that info in a terminal as plain text.